src/input/mod.rs

```diff
@@ -329,34 +329,6 @@ impl Hit for Button {
     }
 }
 
-impl Hit for axis::Key {
```

Comment:
Maybe we should check for `keys_pressed` here and return `true` in case either `pos` or `neg` button was pressed?

Reply:
This is where the semantics of `trait Hit` are unclear. Does `hit` mean hit only in the current frame, or otherwise?

Reply:
hit fir key means "this key has been pressed and it's in pressed state atm

Reply:
In that case this needs to be amended as suggested. Will fix.